source/reviews/pushback.txt

```diff
@@ -0,0 +1,85 @@
+=================================
+
+Sometimes an author will push back on a review. Either they will disagree
+with your suggestion or they will complain that you are being too strict in
+general.
+
+Who Is Right
+------------
+
+When a author disagrees with your suggestion, first take a moment to consider
+if they are correct. Often, they are closer to the content than you are, and so
+they might really have a better insight about certain aspects of it. Does their
+argument make sense? Does it make sense from a content health perspective? If so,
+let them know that they are right and let the issue drop.
+
+However, authors are not always right. In this case the reviewer should
+further explain why they believe that their suggestion is correct. A good
+explanation demonstrates both an understanding of the author's reply, and
+additional information about why the change is being requested.
+
+In particular, when the reviewer believes their suggestion will improve content
+health, they should continue to advocate for the change if they believe the
+resulting content quality improvement justifies the additional work requested.
+**Improving content health tends to happen in small steps.**
+
+Sometimes it takes a few rounds of explaining a suggestion before it really
+sinks in. Just make sure to always stay polite and let the author know that
+you *hear* what they're saying, you just don't *agree*.
+
+Upsetting Authors
+-----------------
+
+Reviewers sometimes believe that the author will be upset if the reviewer
+insists on an improvement. Sometimes authors do become upset, but it is
+usually brief and they become very thankful later that you helped them improve
+the quality of their content. Usually, if you are polite in your comments,
+authors actually don't become upset at all, and the worry is just in the reviewer's
+mind. Upsets are usually more about *the way comments are written* than about
+the reviewer's insistence on content quality.
+
+Cleaning It Up Later
+--------------------
+
+A common source of push back is that authors (understandably) want to get
+things done. They don't want to go through another round of review just to get
+this PR in. So, they say they will clean something up in a later PR or capture it
+in a ticket, and thus you should LGTM *this* PR now. Some authors are very good
+about this, and will immediately write a follow-up PR that fixes the issue.
+However, experience shows that as more time passes after a authors writes the
+original PR, the less likely this clean up is to happen. In fact, unless the
+author does the clean up *immediately* after the present PR, it often never happens.
+This isn't because authors are irresponsible, but because they have a lot of work
+to do and the cleanup gets lost or forgotten in the press of other work. Thus, it
+is usually best to insist that the author clean up their PR *now*, before the
+content is in the corpus and "done." Letting people "clean things up later" is a
```

Comment:
I think these guidelines would work better for the author rather than the reviewer. That way, it doesn't introduce potential conflict and doesn't push blame on the reviewer for not being insistent enough when the result is that it falls through the cracks. If the guidelines were clear to the author, it's clear that it was their responsibility to take care of, and not that the reviewer wasn't insistent enough.